Changes needed in client.cfg, mainly :
    Making use of RabbitMQ (default is ActiveMQ)
    The default port used to connect to RabbitMQ is 61613
    Default user : 'mcollective' & passwd : 'changeme'
    Change RabbitMQ to log to /dev/stderr, so it goes to the SMF log
    Set the PSK type explicitly to UID and the default to 'changeme'

NOTE : This patch is developed in-house (and Solaris specific)

--- marionette-collective-2.8.8/etc/client.cfg.dist.orig	2016-09-19 08:51:01.733783224 -0700
+++ marionette-collective-2.8.8/etc/client.cfg.dist	2016-09-19 08:51:34.482722348 -0700
@@ -1,19 +1,27 @@
+#######################################################################
+# Oracle has modified the originally distributed contents of this file.
+#######################################################################
+
 main_collective = mcollective
 collectives = mcollective
-libdir = /usr/libexec/mcollective
-logger_type = console
 loglevel = warn
+libdir = /usr/share/mcollective
+logger_type = file
+# Send output to SMF service log
+logfile = /dev/stderr
 
 # Plugins
 securityprovider = psk
-plugin.psk = unset
+plugin.psk = changeme
+plugin.psk.callertype = uid
 
-connector = activemq
-plugin.activemq.pool.size = 1
-plugin.activemq.pool.1.host = stomp1
-plugin.activemq.pool.1.port = 6163
-plugin.activemq.pool.1.user = mcollective
-plugin.activemq.pool.1.password = marionette
+connector = rabbitmq
+plugin.rabbitmq.vhost = /mco
+plugin.rabbitmq.pool.size = 1
+plugin.rabbitmq.pool.1.host = localhost
+plugin.rabbitmq.pool.1.port = 61613
+plugin.rabbitmq.pool.1.user = mcollective
+plugin.rabbitmq.pool.1.password = changeme
 
 # Facts
 factsource = yaml
